spec/models/webhooks/utilities_spec.rb

```diff
@@ -35,7 +35,9 @@
     # load dependent models
     module Testing
       include Webhooks::Utilities
-      EVENTS = %w[model_event model_event2].freeze
+      # rubocop:disable Lint/OrAssignmentToConstant
+      EVENTS ||= %w[model_event model_event2].freeze
+      # rubocop:enable Lint/OrAssignmentToConstant
```

Comment:
Should we do this instead?
```suggestion
      EVENTS = %w[model_event model_event2].freeze unless defined? EVENTS
```

Reply:
Sure, it's quite similar. I don't have a preference.